exercises/concept/train-driver/.docs/instructions.md

```diff
@@ -0,0 +1,111 @@
+# Instructions
+
+Your friend is a train driver and has to drive cargo trains between cities.
+Although your friend isn't amazing with handling the computers and would like some help with it.
+Your friend would like your help organizing the train and correcting the mistakes in the data.
+
+
+```exercism/note
+To practice, use a for the rest or spread to solve each of the tasks below.
+```
+
+## 1. Convert the data to an array
+
+Your friend has been keeping track of how much each wagon weighs. Although they are not sure how many wagons and would like the data to be returned as an array.
+
+```exercism/note
+
+Implement a function `getListOfWagons` that accepts an unknown amount of whole numbers that contains the weight of each wagon.
+It should return an array of all the wagon weights.
+
+```
+
+```javascript
+getListOfWagons(5, 7, 12, 3, 14, 8, 3);
+// => [5, 7, 12, 3, 14, 8, 3]
+```
+
+## 2. Move the first two elements to the end of the array
+
+Now that you got a general feel for handling your friend's data. Your friend has noticed that the first two days' values are not in the correct place. 
+Your friend would like you to move the first two days' value to the end of the array.
+
+```exercism/note
+
+Implement a function `fixListOfWagons` that accepts an array of the weight of each wagon.
+It returns an array where the 2 first elements are moved to the end of the array.
```

Comment:
As for the headers, try to write the instructions in the vocabulary of the story, e.g. items -> wagons. The translation of the story to the programming terms is part of what students train here.